src/extensions/default/Microsoft.DotNet.UpgradeAssistant.Steps.Packages/PackageUpdaterStep.cs

```diff
@@ -262,9 +263,27 @@ protected override Task<UpgradeStepApplyResult> ApplyImplAsync(IUpgradeContext c
 
                 _action(file, _operation);
 
+                AddResultToContext(context);
                 return Task.FromResult(new UpgradeStepApplyResult(UpgradeStepStatus.Complete, Title));
             }
 
+            private void AddResultToContext(IUpgradeContext context)
+            {
+                var result = new OutputResult()
+                {
+                    FileLocation = context.CurrentProject?.GetFile()?.FilePath ?? "Project not loaded",
+                    RuleId = Id,
+                    ResultMessage = Title
+                };
+
+                context.Results.Add(new OutputResultDefinition()
```

Comment:
This chunk seems common - could we add a constructor or `.Create()` method on the type to just take `Name` but prepopulate `InformationUri` and `Results` as we continue to do in this PR?